src/test/java/org/cactoos/http/HtBodyTest.java

```diff
@@ -23,12 +23,11 @@
  */
 package org.cactoos.http;
 
-import java.io.IOException;
 import org.cactoos.io.InputOf;
+import org.cactoos.matchers.TextHasString;
```

Comment:
@Vatavuk please use `org.llorllale.cactoos.matchers.TextHasString` (we'll soon be removing `org.cactoos.matchers` from the classpath)